src/lib/assetBridger/erc20Bridger.ts

```diff
@@ -1051,8 +1070,18 @@ export class AdminErc20Bridger extends Erc20Bridger {
         setTokenGas.gasLimit,
         setGatewayGas.gasLimit,
         doubleFeePerGas,
-        setTokenDeposit,
-        setGatewayDeposit,
+        scaleToNativeDecimals({
+          amount: setTokenGas.gasLimit
+            .mul(doubleFeePerGas)
+            .add(setTokenGas.maxSubmissionCost),
+          decimals: nativeTokenDecimals,
+        }),
+        scaleToNativeDecimals({
+          amount: setGatewayGas.gasLimit
+            .mul(doubleFeePerGas)
+            .add(setGatewayGas.maxSubmissionCost),
+          decimals: nativeTokenDecimals,
+        }),
         l1SenderAddress,
       ])
 
```

Comment:
Do we need to set `value` in the return here? I think it only should be set if L3 chains use ETH as the native currency?

Reply:
Updated so that we only pass value when ETH is the native currency